ftw/contacts/contents/contactfolder.py

```diff
@@ -2,12 +2,20 @@
 from plone.dexterity.content import Container
 from plone.supermodel import model
 from zope.interface import implements
+from zope import schema
+from ftw.contacts import _
 
 
 class IContactFolderSchema(model.Schema):
     """A contactfolder type schema interface
     """
 
+    hide_contacts_image = schema.Bool(
+        title=_(u'contactdirectory_hide_contacts_image', default=u'Hide images of contacts'),
+        default=False,
+        description=_(u'If activated, the images of the contacts are not shown.')
```

Comment:
According to our guidelines we use a message id of the form `help_<name_of_the_file>` for the description.